custom_components/resmed_myair/client/myair_client.py

```
@@ -37,9 +38,9 @@ async def connect(self) -> str:
         """Connect to ResMed myAir."""
 
     @abstractmethod
-    async def get_user_device_data(self) -> Mapping[str, Any]:
+    async def get_user_device_data(self, initial: bool | None = False) -> MyAirDevice:
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)